package.json

```diff
@@ -7,7 +7,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.4",
     "@fortawesome/free-solid-svg-icons": "^5.3.1",
     "@fortawesome/react-fontawesome": "^0.1.3",
-    "@material-ui/core": "^3.9.3",
+    "@material-ui/core": "^4.11.0",
```

Comment:
I upgraded the material ui core because I was going to use their tab component. But later decided not to because it will require a lot of overriding the material ui styling.